staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/staticwebhook.go

```diff
@@ -0,0 +1,95 @@
+package generic
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apiserver/pkg/admission/plugin/webhook"
+	"k8s.io/klog"
+)
+
+var (
+	scheme = runtime.NewScheme()
+	codecs = serializer.NewCodecFactory(scheme)
+)
+
+func extractWebhookAccessors(staticConfigFile string, defaultor WebhookDefaultor, validator WebhookValidator) ([]webhook.WebhookAccessor, error) {
+	if staticConfigFile == "" {
+		return []webhook.WebhookAccessor{}, nil
+	}
+
+	data, err := ioutil.ReadFile(staticConfigFile)
+	if err != nil {
+		klog.Fatalf("Couldn't open static configuration file %s: %#v", staticConfigFile, err)
```

Comment:
I expected the logic to load content from the file and default/convert/validate it to return errors rather than exiting the process, so it could be run in the background to pick up changes to the file.

Reply:
Done